docs/graphcast/radios/poi-radio.md

```diff
@@ -119,21 +119,37 @@ We also provide pre-built binaries for Ubuntu and MacOS, which you can find in t
 
 In the configuration table below is the full list of environment variables you can set, along with example values.
 
-
 See [Basic Configuration](#basic-configuration) above. The following environment variables are optional:
 
-| Name                                  | Description and examples                                                                                  |
-| ------------------------------------- | ---------------------------------------------------------------------------------------- |
-| `COLLECT_MESSAGE_DURATION` (Optional) | Seconds that the POI Radio will wait to collect remote POI attestations before making a comparison with the local POI. Example: `120` for 2 minutes. |
-| `WAKU_HOST` (Optional)                | Interface onto which to bind the bundled Waku node. Example: `127.0.0.1` |
-| `WAKU_PORT` (Optional)                | P2P port on which the bundled Waku node will operate. Example: `60000` |
-| `WAKU_NODE_KEY` (Optional)            | Static Waku Node Key. |
-| `BOOT_NODE_ADDRESSES` (Optional)      | Peer addresses to use as Waku boot nodes. Example: `"addr1, addr2, addr3"` |
-| `SLACK_TOKEN` (Optional)              | Slack Token to use for notifications. Example: `xoxp-0123456789-0123456789-0123456789-0123456789` |
-| `SLACK_WEBHOOK` (Optional)            | Slack Webhook to use for notifications. Example: `https://hooks.slack.com/services/<ID>/<ID>/<TOKEN>` |
-| `RUST_LOG` (Optional)                 | Rust tracing configuration. Example: `graphcast_sdk=debug,poi_radio=debug`, defaults to `info` for everything |
-
-`SLACK_TOKEN` and `SLACK_WEBHOOK` are used for POI divergence notifications in a Slack channel.
+| Name                                  | Description and examples                                                                                                                                                                                                           |
+| ------------------------------------- | ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `MNEMONIC` (Optional)                 | Mnemonic to the Graphcast ID wallet (first address of the wallet is used; Only one of `PRIVATE_KEY` or `MNEMONIC` is needed). Example: `claptrap armchair violin...`                                                               |
```

Comment:
We probably don't need `(Optional)` for all the variables since it is written on L122. Maybe to stay explicitly clear we rename the column to be `| Name (Optional variables) ...`

Reply:
this should be done